Go Client.Tests/GoRules_Tests.cpp

```diff
@@ -21,15 +21,15 @@ TEST(Rules, Stone_dosent_die_when_sorounded_by_its_own_stones)
 {
 	GoBoard testBoard(3);
 	board expectedboard =
-	{ { empty, black, empty },
-	{ black, black, black },
-	{ empty, black, empty } };
+	{ { stone::EMPTY, stone::BLACK, stone::EMPTY },
+	{ stone::BLACK, stone::BLACK, stone::BLACK },
+	{ stone::EMPTY, stone::BLACK, stone::EMPTY } };
 	Rules testrules(testBoard);
-	testBoard.PlayStone(0, 1, black);
-	testBoard.PlayStone(1, 1, black);
-	testBoard.PlayStone(1, 0, black);
-	testBoard.PlayStone(1, 2, black);
-	testBoard.PlayStone(2, 1, black);
+	testBoard.PlayStone(0, 1, stone::BLACK);
```

Comment:
you need a function to convert an ASCII board into a game board in your system... this isn't visual enough ;)

a multi-line string with Xs and Os would do.... or Bs and Ws for black and white...